conf/eddie.config

```diff
@@ -1,13 +1,12 @@
 //Profile config names for nf-core/configs
 params {
     config_profile_description = 'University of Edinburgh (eddie) cluster profile provided by nf-core/configs.'
-    config_profile_contact = 'Alison Meynert (@ameynert)'
+    config_profile_contact = 'Graeme Grimes'
```

Comment:
Can we have a GitHub handle for pinging?

Reply:
sure it's @ggrimes